docs/CHANGELOG.rst

```diff
@@ -3,6 +3,17 @@ Version history
 
 We follow `Semantic Versions <https://semver.org/>`_.
 
+0.8.1 (15.11.24)
+*******************************************************************************
+- Restore ability to specify ``item_class`` in ``ListComponent``. This is
+  because getting ``item_class`` from the first generic variable may become
+  invalid when inheriting and adding new generic variables in component.
```

Comment:
Could you add an example of such a case? For future reference? Maybe even add it to docs?